### ISSUE TITLE
Crash when dragging tabs

### ISSUE THREAD

**Issue by Yanstory** [stefankueng/BowPad#170]
Actually is: If the file path contains non-ASCII words, and you drag it out of the window to separate the window, the crash triggered.

**Comment by wmjordan** [comment]
I have quite a few files with Chinese in their file names and folder names.

I could not reproduce this. I opened two files with Chinese characters in their file names or paths, dragged the document tab and the file was opened in new BowPad window. Everything was fine.
It seemed that the non-ASCII characters were not causing the problem...

**Comment by Yanstory** [comment]
Yes it is. I just retry it with all-ASCII path and it reproduced, even with portable or installed version.
the only thing I found is an "successful" window separating cause a new window with a path contains "temp", and the tab name is very long.

**Comment by wmjordan** [comment]
Would you like to try the newest version compiled by me (BowPad.xml and LexLog.cxx are modified to suite my own needs, the rest is just the newest source code)?
If you'd like to give it a try I can post it here.

**Comment by Yanstory** [comment]
I'll have a try.

**Comment by wmjordan** [comment]
OK, let's see whether the newest version works on your computer.

[BowPad.zip](https://github.com/stefankueng/BowPad/files/6433122/BowPad.zip)

**Comment by Yanstory** [comment]
Yes it works fine now, the tab in new window shows the correct file name.

**Comment by wmjordan** [comment]
Good.

**Comment by wmjordan** [comment]
I will close this issue if nothing goes wrong with the latest build.

**Comment by stefankueng** [comment]
@wmjordan  could you tell us what changes you've made to BowPad.xml and LexLog.cxx? Maybe I can include them in BP or implement an option/configuration so you don't have to maintain your own versions.

**Comment by wmjordan** [comment]
Thank you Stefan!
There's nothing wonderful in my home-brew version actually. I guess we'd better leave it along.

LexLog: I thought the numeric highlight was unimportant to me and my log files had a lot of date-time and HexBin representations, the numeric highlight could make my log files look too colorful but those colors in HexBin were useless for me, thus I commented all numeric parsing and highlighting code to save some CPU resource.

BowPad.xml: It was about the issue #133 that the "Other" tab took sometime to show and consumed a bit too much RAM (about 150MB extra RAM consumption on my computer). After consulting you, we addressed that it was the font box which read all fonts installed in the system and generated previews for them, that was built in the Ribbon framework and could not be fixed, which caused the tab to take a few seconds to show on my computer which had hundreds of font files. I hereby commented the font group in the Ribbon definition.
